spec/rails_helper.rb

```diff
@@ -12,6 +12,7 @@
 require "webmock/rspec"
 WebMock.disable_net_connect!(allow_localhost: true,
                              allow: "chromedriver.storage.googleapis.com")
+# WebMock.enable_net_connect!
```

Comment:
???

Reply:
This was to allow the testing of http requests in the project show rspec